dbt/tests/adapter/dbt_debug/test_dbt_debug.py

```diff
@@ -85,11 +85,11 @@ def test_empty_target(self, project):
         self.assertGotValue(re.compile(r"\s+output 'none_target'"), "misconfigured")
 
 
-class TestDebugProfileVariablePostgres(BaseDebugProfileVariable):
+class TestDebugPostgres(BaseDebugPostgres):
```

Comment:
Whats up with this rename? I'm missing something

Reply:
The diffs on this PR are confusing.

`BaseDebugProfileVariable` has no test methods in it, which means that `TestDebugProfileVariablePostgres` did nothing. So I removed it.

Also, I broke out `TestDebugPostgres` into a base class and a test class, which got caught up in this.

So it went from this:
```python
class TestDebugPostgres(BaseDebug):
    def test_ok(self, project):
        ...


class TestDebugProfileVariablePostgres(BaseDebugProfileVariable):
    pass  # this class contains no test methods


class TestDebugInvalidProjectPostgres(BaseDebug):
    def test_empty_project(self, project):
        ...
```

to this:
```python
class BaseDebugPostgres(BaseDebug):
    def test_ok(self, project):
        ...


# break base class out from test class
class TestDebugPostgres(BaseDebugPostgres):
    pass


# delete empty test class TestDebugProfileVariablePostgres


class BaseDebugInvalidProjectPostgres(BaseDebug):
    def test_empty_project(self, project):
        ...


# break base class out from test class
class TestDebugInvalidProjectPostgres(BaseDebugInvalidProjectPostgres):
    pass
```

Reply:
Yep that makes sense. Just wanted to double check.